### PR TITLE
fix: policy route using wrong name/param

### PR DESCRIPTION
**Pull request by kleinfreund**
Fixes a small issue with links to specific policy views using the route name for the general policies route and a `policyType` param which doesn’t exist.

Signed-off-by: Philipp Rudloff <philipp.rudloff@konghq.com>

### CODE REVIEW

**Review by johncowen — APPROVED** [review]
This would be good in the upcoming release right? ( @Automaat FYI. Not sure if its too late if this is something that would be good to go in the upcoming release)

**Comment by kleinfreund** [comment]
> This would be good in the upcoming release right? (@Automaat FYI. Not sure if its too late if this is something that would be good to go in the upcoming release)

I think this is such a minor issue that it doesn’t necessarily need to be included in the release of 2.1.0. In any case, it’s also not difficult to do as long as we haven’t done more than prepare the release branch in Kuma. We can merge it which would automatically propagate it to Kuma’s `master` branch from where it will have to be cherry-picked into the `release-2.1` branch. Leaving this up to @Automaat as the release manager to decide.

**Pull request merged**
